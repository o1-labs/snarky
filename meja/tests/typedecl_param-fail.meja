type t(int, 'a) = 'a;
