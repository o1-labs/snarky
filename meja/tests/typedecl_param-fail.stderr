File "tests/typedecl_param-fail.meja", line 1, characters 7-10:
Error: Syntax error: Expected a type parameter, but got int.
